src/renderer/api/kube-watch-api.ts

```diff
@@ -41,13 +40,13 @@ export interface IKubeWatchLog {
 
 @autobind()
 export class KubeWatchApi {
-  private cluster: Cluster;
-  private namespaceStore: NamespaceStore;
-
   private requestId = 0;
-  private isConnected = false;
   private reader: ReadableStreamReader<string>;
-  private subscribers = observable.map<KubeApi, number>();
+
+  @observable.ref private getCluster: () => Cluster;
+  @observable.ref private getNamespaces: () => string[];
```

Comment:
These don't have an initial value so they should be marked as optional

```suggestion
  @observable.ref private getCluster?: () => Cluster;
  @observable.ref private getNamespaces?: () => string[];
```

Reply:
This was not changed or commented on, unresolving.

Reply:
fixed, PTAL